packages/@icon-magic/icon-models/src/flavor.ts

```diff
@@ -38,25 +39,33 @@ export class Flavor extends Asset {
       // create a an empty map
       this.types = new Map();
     }
+    this.flavorConfig = this.configureConfig();
   }
 
   /**
-   * returns  flavor data that needs to be stored in the config file
+   * Populates FlavorConfig with flavor data that needs to be stored and
+   * @returnsobject to be set as the config
```

Comment:
space between `@returns` and `object`. also, this method should have a return type.